samples/nrf9160/serial_lte_modem/README.rst

```diff
@@ -141,6 +141,22 @@ The following proprietary GPS AT commands are used in this sample:
 
 * AT#XGPS=<op>[,<mask>]
 
+MQTT AT Commands
+****************
+
+The following proprietary MQTT AT commands are used in this sample:
+
+* AT#XMQTTCON=<op>[,<url>,<port>[,<sec_tag>]]
+* AT#XMQTTPUB=<topic>,<datatype>,<msg>,<qos>,<retain>
+* AT#XMQTTSUB=<topic>,<op>[,<qos>]
+
+The following unsolicited notification reports the MQTT events.
+* #XMQTTEVT=<type>,<result>
+
+The following unsolicited notification reports the received publish message. The topic and message
+will then be reported.
```

Comment:
```suggestion
The following unsolicited notification indicates that a publish message was received and reports the topic and the message:
```

And does it report the topic and the message, or the topic length and the message length?

Reply:
The notification will first notify the data type, topic length and message length. Then the topic and message is notified. Carriage Return and Line Feed is inserted before topic and message. The description becomes:

The following unsolicited notification indicates that a publish message was
received and reports the topic and the message:

* #XMQTTMSG=<datatype>,<topic_length>,<message_length><CR><LF><topic><CR><LF><message>